packages/search/src/search.tsx

```diff
@@ -395,36 +393,35 @@ const InnerSearch = (
 		return null;
 	};
 
-	const renderReskinSearchIcon = () => {
-		const searchIcon = (
-			<svg viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
-				<path
-					fillRule="evenodd"
-					clipRule="evenodd"
-					d="M13.8269 19.8984L8.49362 24.5651L7.50586 23.4362L12.8392 18.7695L13.8269 19.8984Z"
-				/>
-				<path
-					fillRule="evenodd"
-					clipRule="evenodd"
-					d="M17.9994 21.1667C21.221 21.1667 23.8327 18.555 23.8327 15.3333C23.8327 12.1117 21.221 9.5 17.9994 9.5C14.7777 9.5 12.166 12.1117 12.166 15.3333C12.166 18.555 14.7777 21.1667 17.9994 21.1667ZM17.9994 22.6667C22.0494 22.6667 25.3327 19.3834 25.3327 15.3333C25.3327 11.2832 22.0494 8 17.9994 8C13.9493 8 10.666 11.2832 10.666 15.3333C10.666 19.3834 13.9493 22.6667 17.9994 22.6667Z"
-				/>
-			</svg>
-		);
-
-		return <Icon icon={ searchIcon } size={ 32 } className="search-component__icon-search" />;
-	};
-
 	const renderOpenIcon = () => {
 		const enableOpenIcon = pinned && ! isOpen;
 
+		const defaultSearchIcon = (
```

Comment:
The bundle size has somehow bloated, and the only suspicious place I'd guess is this. I'm likely wrong though 🤔

Either way, I'm wondering if it would be better to extract this as a individual component we can import like https://github.com/Automattic/wp-calypso/blob/trunk/client/landing/stepper/declarative-flow/internals/steps-repository/hundred-year-plan-step-wrapper/hundred-year-plan-logo.tsx#L1. Otherwise, given no other hidden optimization, this `defaultSearchIcon` will be recreated every time `renderOpenIcon` is called .

Reply:
> and the only suspicious place I'd guess is this.

Interesting! Is there a reason that you suspected this to be the culprit. It was simply a renaming of the function name and variable name.

Reply:
I actually opted to use the already treeshaken search which helped here and looks fine IMO
https://github.com/Automattic/wp-calypso/pull/99572/commits/ce1b8244a7728f46741eeb73c1101948ca0ea54b

Reply:
Nice finding! Regarding the above reasoning, it was simply a guess so I was likely wrong. Now that you've reused the existing component, it's no longer an issue :)